docs/gcp_kms.md

```diff
@@ -0,0 +1,190 @@
+---
+id: gcp_kms
+title: CloudKMS
+---
+
+##
+**Google Cloud Platform configuration**
+
+Create a new project or use an existing project and the service accounts used with Signatory should have the following permissions. It may be achieved by using custom roles (see [https://console.cloud.google.com/iam-admin/roles](https://console.cloud.google.com/iam-admin/roles)) \
+Project name is required in the signatory config.
+
+###
+**Basic permissions**
+* `cloudkms.cryptoKeyVersions.get`
+* `cloudkms.cryptoKeyVersions.list`
+* `cloudkms.cryptoKeyVersions.viewPublicKey`
+* `cloudkms.cryptoKeys.get`
+* `cloudkms.cryptoKeys.list`
+
+###
+**Sign**
+* `cloudkms.cryptoKeyVersions.useToSign`
+###
+**Import**
+
+* `cloudkms.cryptoKeyVersions.create`
+* `cloudkms.cryptoKeys.create`
+* `cloudkms.importJobs.create`
+* `cloudkms.importJobs.get`
+* `cloudkms.importJobs.list`
+* `cloudkms.importJobs.useToImport`
+
+###
+**Configuration parameters \
+Below are the configuration fields which are required for Signatory.**
+
+<table>
+  <tr>
+   <td>
+<strong>Name</strong>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>application_credentials
+   </td>
+   <td>string
+   </td>
+   <td>
+   </td>
+   <td>Path to the GCP application token JSON file (overrides <code>GOOGLE_APPLICATION_CREDENTIALS</code> environment variable)
+   </td>
+  </tr>
+  <tr>
+   <td>application_credentials_data
+   </td>
+   <td>string
+   </td>
+   <td>
+   </td>
+   <td>GCP application token JSON data (overrides <code>application_credentials</code>)
+   </td>
+  </tr>
+  <tr>
+   <td>project
+   </td>
+   <td>string
+   </td>
+   <td>✅
+   </td>
+   <td>Project name
+   </td>
+  </tr>
+  <tr>
+   <td>location
+   </td>
+   <td>string
+   </td>
+   <td>✅
+   </td>
+   <td>Location
+   </td>
+  </tr>
+  <tr>
+   <td>key_ring
+   </td>
+   <td>string
+   </td>
+   <td>✅
+   </td>
+   <td>Key ring name
+   </td>
+  </tr>
+</table>
+
+###
+
+**Key Management**
+
+Under <code>key management</code> create a new <code>key-ring</code> with any location and create a key with <code>purpose</code> as<strong> <code>Asymmetric sign </code></strong>and<code> protection level </code>as<code>HSM.</code>
```

Comment:
For code sections I think you can just use the backticks like in the application access section below. I don't know if mixing in HTML stuff in markdown is the best long term